design/fast_finality.md

```diff
@@ -0,0 +1,276 @@
+# Fast finality design
+
+## Scope
+
+The introduction of a single sequencer to improve cost-per-transaction during the bootstrapping phase, as 
+described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md).
+
+## Glossary
+
+* *Light batch*: A set of transactions produced by the designated sequencer, considered to be soft-finalised in that 
+  order
+
+## Requirements
+
+* Finality
+  * Transaction *soft* finality (finality guaranteed by the sequencer) is achieved in under one second
+  * There is eventual transaction *hard* finality (finality guaranteed by the L1)
+  * The sequencer is incentivised to achieve hard finality on an agreed cadence
+  * The sequencer is incentivised to hard-finalise transactions in the same order they are soft-finalised
+  * The sequencer is not able to "rewrite history" (or is strongly disincentivised from doing so), even for soft-final
+    transactions
+* Censorship resistance
+  * End-users can bypass the sequencer to include transactions on the L1 directly (possibly at higher cost and slower 
```

Comment:
Yes, sorry. This was based on my old understanding of the inbox.

I'm going to leave the "possibly" because the requirement isn't for this mechanism to necessarily be more expensive.